docs/application/vscode-ext/web.md

```diff
@@ -63,87 +63,142 @@ If you do not have a real device, you can run your application in a virtual envi
 
 This section explains how to use VS Code Extension for Tizen Web to develop applications.
 
+### Change Tizen SDK Path
+
+To change a Tizen SDK path, follow these steps:
+
+1. In VS Code, click **View** > **Command Palette**.
+2. In the input field of the **Command Palette** that appears, type **Tizen Web: Change Tizen SDK Path** and press **Enter**.
+3. The change notification appears. If you do not want to change the path, click **Keep**. If you click **Change Path**, In the input field of the Command Palette that appears, type change to path and press **Enter**. 
+
+    > **Default Path**
+    >
+    > Window OS : `c:\tizen-studio`
+    >
+    > Linux OS : `/home/username/tizen-studio`
+
+    ![Change Tizen SDK Path](media/tizen_web24.png)
+
+### Install Tizen Baseline SDK
+
+> **Note**
+>
+> You must install Oracle Java Development Kit (JDK) 8 or OpenJDK 12 for Tizen Studio 3.5 and higher.
+>
+> [Oracle Java Download page](https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)
+
+To install a Tizen Baseline SDK, follow these steps:
+
+1. In VS Code, click **View** > **Command Palette**.
+2. In the input field of the **Command Palette** that appears, type **Tizen Web: Install Tizen Baseline SDK** and press **Enter**.
+3. If SDK is already installed, click **Yes**, then run **Tizen Web: Change SDK Path**.
+
+    ![install check](media/tizen_web31.png)
+4. Click **No**, To perform a fresh installation and begin the Tizen Baseline SDK installation, click **Install**. If the path of the popup is **null**, change the path by clicking **Change Path**.
+
+    ![install ask](media/tizen_web32.png)
+5. Proceed with the installation. The installation progress appears in the log. The extension reloads when the installation is complete.
```

Comment:
Do we have steps to be performed while proceeding with the installation? if not, then please add this as a step result for step 4.

Reply:
done.